### DISCUSSION TITLE
Have the same font for channel names in tabs

### DISCUSSION THREAD

**Discussion by PepeLUL** [Chatterino/chatterino2#4137] [Ideas]
Fonts for channel names should be the same as you changed the font to
for example ive changed my font to Roboto here and is reflected in the channel tab instead of the default font
![chatterino_eopfUvMJuU](https://user-images.githubusercontent.com/19637313/201220001-2f8f4bd5-73d4-49dc-949a-7d1d0fdd042a.png)

**Comment by pajlada** [comment]
Could you highlight in a big red circle what text you'd like the font setting to apply to?

**Reply by PepeLUL** [reply]
![circle](https://user-images.githubusercontent.com/19637313/201496125-8cba1a09-3128-48ab-8d72-40b01f0fa9cb.png)
